test/e2e/serial/workload_placement_test.go

```diff
@@ -158,7 +158,7 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 				Expect(err).ToNot(HaveOccurred(), "cannot detect the zone ID from %q", zone.Name)
 				// now put a minimal pod (1 cpu 1Gi) on both zones. Now the target node as whole will still have the
 				// required resources, but no NUMA zone individually will
-				targetedPaddingPod := objects.NewTestPodPause(fxt.Namespace.Name, fmt.Sprintf("tgtpadpod-%s", zone.Name))
+				targetedPaddingPod := objects.NewTestPod(fxt.Namespace.Name, fmt.Sprintf("tgtpadpod-%s", zone.Name))
```

Comment:
renaming this function is unnecessary - and I'd said wrong in this context

Reply:
thanks for your comments.
why do we want to limit the use only for Pause image in the name of the function?

Reply:
Because I think is not time yet to generalize the function. We still want a container which does `pause` and sits idle

Reply:
ok, I see. updated. thanks!